src/ProfilesList.json

```diff
@@ -173,6 +173,7 @@
   "dpass47.json",
   "abdosahsah.json",
   "athenacats.json",
+  "hackerspider09",
```

Comment:
add `.json` at last of the filename The filename should be `hackerspider09.json`